docs/source/managecontainers.rst

```diff
@@ -37,10 +37,10 @@ Manage Containers
 
 :doc:`port <markdown/podman-port.1>` List port mappings or a specific mapping for the container
 
-:doc:`restart <markdown/podman-restart.1>` Restart one or more containers
-
 :doc:`prune <markdown/podman-container-prune.1>` Remove all stopped containers
 
+:doc:`restart <markdown/podman-restart.1>` Restart one or more containers
+
```

Comment:
should line 40 and 44 be higher up?  'podman-container-*'

Reply:
I'm sorting on the subcommand name, not the link target. 'prune' < 'restart'

Reply:
(does that address your concern?)

Reply:
My bad, I was liking at the link targets there.